react/typings/globals.d.ts

```diff
@@ -33,7 +33,7 @@ declare global {
     skuName: string
     skuSpecifications: SKUSpecification[]
     options?: AssemblyOptionInput[]
-    uniqueId: string
+    uniqueId?: string
```

Comment:
In which scenario is it ok to have an undefined uniqueId?

Reply:
uniqueIds doesn't actually exists in the `CatalogItem` type, that's why I moved it to optional because someone may be passing them to `addItems`